cli/tests/integration/lsp_tests.rs

```diff
@@ -3442,7 +3525,10 @@ fn lsp_diagnostics_refresh_dependents() {
       "version": 1
     })
   );
-  client
+
+  // todo(dsherret): I'm not sure exactly what this is testing here
```

Comment:
I should resolve this todo, but I'm not really sure what was being tested before. I think previously just a lot of past diagnostics were buffered up because each `did_open` actually causes 3 * number of files open, but only 3 was being read each time.

Reply:
That is trying to be tested here is that the diagnostics for dependents get invalidated and recalculated.

For example if you have `a.ts` the exports something which is imported into `b.ts`, when you modify `a.ts`, it impacts that diagnostics of `b.ts` without `b.ts` being edited.  We had issues where diagnostics were hanging around until you actually touched/modified a dependent file.